config.yml

```diff
@@ -0,0 +1,46 @@
+defaults:
+  taskcluster:
+    rootUrl:        https://taskcluster-events-staging.herokuapp.com/ 
+    credentials:
+      clientId:     !env TASKCLUSTER_CLIENT_ID
+      accessToken:  !env TASKLCSUTER_ACCESS_TOKEN
+  
+  server:
+    port:           !env:number PORT
+    env:            !env NODE_ENV
+    forceSSL:       !env:flag FORCE_SSL
+    trustProxy:     false
+  
+  monitor:
+    mock:           true
+  
+  pulse:
+    username:       !env PULSE_USERNAME
+    password:       !env PULSE_PASSWORD
+    hostname:       !env PULSE_HOSTNAME
+    vhost:          /
+
+
+production:
+  server:
+    env:            production
+    forceSSL:       true
```

Comment:
This should also be configurable in production. I know 99% of our services hardcode this currently but I'm trying to get them to stop too :p